reporters/index.js

```diff
@@ -138,11 +146,29 @@ const Poll = async function(callback) {
     // UNISWAP_PRICE_FEED_CONFIG={"type":"uniswap","twapLength":86400,"lookback":7200,"invertPrice":true,"uniswapAddress":"0x1e4F65138Bbdb66b9C4140b2b18255A896272338"}
     const uniswapPriceFeedConfig = JSON.parse(process.env.UNISWAP_PRICE_FEED_CONFIG);
 
-    // Change `periodLengthSeconds` to modify how far back the report will look for its shorter report period. For example, setting this to
-    // `24 * 60 * 60` means that the report will include aggregated data for the past 24 hours.
-    const periodLengthSeconds = process.env.PERIOD_REPORT_LENGTH ? process.env.PERIOD_REPORT_LENGTH : 24 * 60 * 60;
-
-    await run(empAddress, walletsToMonitor, referencePriceFeedConfig, uniswapPriceFeedConfig, periodLengthSeconds);
+    // The report will always display "cumulative" and "current" data but it will also show data for a shorter period ("period") whose
+    // start and end dates we can control:
+
+    // Change `endDateOffsetSeconds` to modify the end date for the "period". End date will be (now - endDateOffsetSeconds).
+    const endDateOffsetSeconds =
+      process.env.PERIOD_END_DATE_OFFSET && process.env.PERIOD_END_DATE_OFFSET > 0
+        ? Math.round(process.env.PERIOD_END_DATE_OFFSET)
+        : 0;
```

Comment:
Aren't environment variables strings? Does `> 0` work on a string? Same question below.

Reply:
Ah yes, I'll just `parseInt` it and if that fails the root `try-catch` will report it